backend/template.yaml

```
@@ -214,6 +325,16 @@ Parameters:
   SalesforceInstanceUrl:
     Type: String
     Description: Salesforce instance URL (e.g., https://yourcompany.my.salesforce.com)
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)